api/src/main/java/org/openmrs/api/db/hibernate/HibernateMedicationDispenseDAO.java

```diff
@@ -41,17 +41,14 @@ public MedicationDispense getMedicationDispense(Integer medicationDispenseId) {
 
 	@Override
 	public MedicationDispense getMedicationDispenseByUuid(String uuid) {
-		return sessionFactory.getCurrentSession()
-			.createQuery("select md from MedicationDispense md where md.uuid = :uuid", MedicationDispense.class)
-			.setParameter("uuid", uuid)
-			.uniqueResult();
+		return HibernateUtil.getUniqueEntityByUUID(sessionFactory, MedicationDispense.class, uuid);
 	}
 
 	@Override
 	public List<MedicationDispense> getMedicationDispenseByCriteria(MedicationDispenseCriteria criteria) {
 		CriteriaBuilder criteriaBuilder = sessionFactory.getCurrentSession().getCriteriaBuilder();
-		CriteriaQuery<MedicationDispense> criteriaQuery = criteriaBuilder.createQuery(MedicationDispense.class);
-		Root<MedicationDispense> md = criteriaQuery.from(MedicationDispense.class);
+		CriteriaQuery<MedicationDispense> cq = criteriaBuilder.createQuery(MedicationDispense.class);
```

Comment:
What is the motivation for changing the variable name from `criteriaQuery` to `cq`?

Reply:
In one of the other PRs you mentioned a few options on the naming conventions for these and we had been using cq in the other PRs

Reply:
Very true indeed! But the challenge here is that the author of this code used `criteriaQuery` which looks like a better variable name than `cq`. So when they notice that we are changing from a clearer name, don't you think that they will feel like we are making their code less readable? 😊

Reply:
Yea that makes sense, I can roll it back